pkg/storage/test/tuples.go

```diff
@@ -674,6 +674,105 @@ func ReadStartingWithUserTest(t *testing.T, datastore storage.OpenFGADatastore)
 	})
 }
 
+func ReadTest(t *testing.T, datastore storage.OpenFGADatastore) {
+	ctx := context.Background()
+
+	tuples := []*openfgav1.TupleKey{
+		tuple.NewTupleKey("document:1", "reader", "user:anne"),
+		tuple.NewTupleKey("document:1", "reader", "user:bob"),
+		tuple.NewTupleKey("document:1", "writer", "user:bob"),
+	}
+
+	storeID := ulid.Make().String()
+
+	err := datastore.Write(ctx, storeID, nil, tuples)
+	require.NoError(t, err)
+
```

Comment:
Do we want to add a test to read all tuples?

Reply:
Ah yes, good call. Let's please add this test.

Reply:
TIL /read supports not passing any tuple key 😆  i'll add that test.